### PR TITLE
IGNITE-22696 Added libs to opensensus to expose prometheus metrics

### PR DESCRIPTION
**Pull request by makalis**
Added opencensus-exporter-stats-prometheus and simpleclient_httpserver to opensensus module to make prometheus metrics work from the box without building any extra libs.

Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [x] There is a single JIRA ticket related to the pull request. 
- [x] The web-link to the pull request is attached to the JIRA ticket.
- [x] The JIRA ticket has the _Patch Available_ state.
- [x] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [x] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [x] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/opencensus/pom.xml [hunk 1]
```diff
@@ -100,14 +100,12 @@
             <groupId>io.opencensus</groupId>
             <artifactId>opencensus-exporter-stats-prometheus</artifactId>
             <version>${opencensus.version}</version>
-            <scope>test</scope>
```
Comment (nva) [inline modules/opencensus/pom.xml:103]: Can we use the runtime scope for exporters like zipkin, prometheus?

Comment (makalis) [reply modules/opencensus/pom.xml:103]: Yes, changed the scope, thanks. Libs not needed for compilation, only for runtime.

Comment (nva) [reply modules/opencensus/pom.xml:103]: But what about zipkin exporter can we fix it too?

Comment (nva) [reply modules/opencensus/pom.xml:103]: But what about zipkin exporter can we fix it too?

Comment (makalis) [reply modules/opencensus/pom.xml:103]: For now opencensus-exporter-trace-zipkin scope is test in AI, and my goal is to make metrics work, not traces. So I suggest to leave it as is for now for it.

**Review by nva — APPROVED** [review]

**Comment by nva** [comment]
@makalis LGTM. Thanks for contribution!

**Comment by nva** [comment]
https://mtcga.gridgain.com/pr.html?serverId=apache&suiteId=IgniteTests24Java8_RunAll&branchForTc=pull/11426/head&action=Latest

**Pull request merged**